native/src/CMakeLists.txt

```diff
@@ -224,6 +225,7 @@ source_group("nova" FILES ${NOVA_HEADERS} ${NOVA_NO_COMPILE} ${NOVA_SOURCE})
 
 add_library(nova-renderer SHARED ${NOVA_SOURCE} ${SPIRV_CROSS_SOURCE})
 set_target_properties(nova-renderer PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
+target_compile_definitions(nova-renderer PUBLIC DUMP_NOVA_CALLS)
```

Comment:
might be a good idea to have it enabled only in DEBUG build

Reply:
Fixed